This is needed since initgroups belongs to NET group
so when NET is disabled in eglibc build then it reports
as undefined symbol

Signed-off-by: Khem Raj <raj.khem@gmail.com>
Upstream-Status: Pending

Index: libc/nss/getent.c
===================================================================
--- libc.orig/nss/getent.c	2012-03-09 09:41:57.099581559 -0800
+++ libc/nss/getent.c	2012-03-09 09:42:13.095582334 -0800
@@ -898,7 +898,7 @@
 D(group)
 D(gshadow)
 DN(hosts)
-D(initgroups)
+DN(initgroups)
 DN(netgroup)
 DN(networks)
 D(passwd)
